app/lib/actions/katello/repository/clone_to_version.rb

```diff
@@ -20,6 +20,8 @@ def plan(repository, content_view_version, incremental = false)
               plan_action(Repository::CloneDockerContent, repository, new_repository)
             elsif new_repository.ostree?
               plan_action(Repository::CloneOstreeContent, repository, new_repository)
+            elsif new_repository.file?
+              plan_action(Repository::CloneFileContent, repository, new_repository)
```

Comment:
I think you need this logic here too:  https://github.com/Katello/katello/blob/master/app/lib/actions/katello/repository/clone_to_environment.rb#L23